react/features/base/util/loadScript.web.js

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
OK. Do you mean that you'll remove it from this PR? I don't think it's necessary here but I'm merely asking to better understand your proposal.

The biggest problem that I have this file/function though is that there's one like it in lib-jitsi-meet. I feel like we should reuse it and have 1 in the 2 projects. What do you think?